stdlib/public/runtime/Metadata.cpp

```diff
@@ -4421,8 +4421,9 @@ static SimpleGlobalCache<ExtendedExistentialTypeCacheEntry,
                          ExtendedExistentialTypesTag>
   ExtendedExistentialTypes;
 
+SWIFT_RUNTIME_EXPORT
 const ExtendedExistentialTypeMetadata *
-swift::swift_getExtendedExistentialTypeMetadata_unique(
+swift_getExtendedExistentialTypeMetadata_unique(
```

Comment:
This doesn't make any sense.  This is causing the declaration to not be matched against the previous declaration in the header, and I don't see anything about this specific function which is different from all of the other functions in the header / implementation.

Reply:
Yeah, looking at the test failure, it looks like the test is running against an installed runtime in the OS and probably just needs to be disabled in that configuration, since the OS runtime will not provide this functionality.